### ISSUE TITLE
Looking for contributions?

### ISSUE THREAD

**Issue by harrisj** [AwkwardTurtle42/TroikaDiscordBot#1]
I am interested in building out the excellent start you have here, but I was wondering if you would want me to send PRs back to this project to merge or if you aren't really interested in maintaining it (I'd work on my fork and credit you). Whatever works for you, I wanted to check.

**Comment by natehole** [comment]
@harrisj I've played around with this project a little bit and written a bot command to generate a character. Let me know if you want it. Right now it's in a dirty development branch with all of the original code commented out, but I could put it into my local master and create a PR for your fork.

**Comment by harrisj** [comment]
Nice, I wonder if we could pool our efforts? I mainly want to be able to do things like tests and some of the things like the Oops table!

**Comment by AwkwardTurtle42** [comment]
Hey, honestly my code here is extremely jank and I even ran into some issues with it last time I tried to use it myself. If either of you want to take it, pull it into shape, use it as a base, or just throw it in the trash you're more than welcome to.

Feel free to fork it off, or I can add you as an owner probably? Honestly not very familiar with github, I'm not a coder by trade so I only know just enough to cause problems.

**Comment by harrisj** [comment]
No self-deprecation needed. We all need to start somewhere and you started on this first! I guess it sounds liike @natehole might be interested in being the maintainer of this code if you didn't want to (or we could be collaborators on this repo if you wanted to let us). Basically, it's good to have a few people to review PRs and the like

**Comment by AwkwardTurtle42** [comment]
Haha, I'm happy to stay involved, but I realistically won't be able to offer much to the effort. If you hadn't commented here I would likely have never looked at this code again.

**Comment by harrisj** [comment]
Fair enough, I appreciate the honest assessment! So it seems like we have a few options. Natehole, do you want to be the primary maintainer on your fork? AwkwardTurtle42, do you want us to be collaborators on this repository and manage all thr contributions here? I'm fine with either option

**Comment by natehole** [comment]
@harrisj I added you as a contributor on my fork. If @AwkwardTurtle42, you would like to add either of us as contributors to your project you can and we can work off of that or work can simply be done off of a fork. It doesn't matter at all to me. Let me know what you think. In the meantime I am going to add some TODO issues on the fork for now.

**Comment by AwkwardTurtle42** [comment]
I can add you both as contributors if you'd like, whatever gives you the most flexibility. No pressure, if it's easier to just work off of a fork go ahead. 

I also have no idea how to actually add contributors, so if you want to be added that way you'll have to point me int he right direction.
